sensor_test.py

```diff
@@ -195,10 +195,10 @@ def test_set_empty_sensor_name(get_sensor_info, set_sensor_name):
     sensor_response = set_sensor_name(" ")
```

Comment:
The value passed to `set_sensor_name()` should be `""`, not an `" "` because the second one isn't actually an empty string because it contains a space.